client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraph.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="provenance-graph-container" v-if="isOpen">
+    <aside slot="left">
+      <div class="btn-group btn-group-sm">
+        <button
+          class="btn btn-secondary"
+          @click="onClickCondensed"
+          > Condensed
+        </button>
+        <button
+          class="btn btn-secondary"
+          @click="onClickExpanded"
+          > Expanded
+        </button>
+      </div>
+    </aside>
+    <close-button @close="onClose"/>
+    <div class="flex-grow-1 position-relative overflow-scroll hide-scrollbar panel-body">
+      <slot name="content"/>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+  import Component from 'vue-class-component';
+  import Vue from 'vue';
+  import { Prop } from 'vue-property-decorator';
+  import CloseButton from '@/components/widgets/CloseButton.vue';
+
+  const components = {
+    CloseButton,
+  };
+
+  @Component({ components })
+  export default class ProvenanceGraph extends Vue {
+    @Prop({ default: false })
+    isOpen: boolean;
```

Comment:
This doesn't need to be a prop if it's never used elsewhere.
The `v-if="isOpen"` can be handle by the parent component directly.

Reply:
Thanks, I've removed the prop.